daemons/gptp/gptp_cfg.ini

```diff
@@ -31,19 +31,11 @@ syncReceiptThresh = 8
 
 [eth]
 
-# PHY delay GB TX in nanoseconds
+# PHY delay GB TX/RX in nanoseconds
```

Comment:
I can change the .ini file we are using.  I was more concerned that others may be caught unaware by this change, as it does break backwards-compatibility as written.